src/algokit/core/tasks/mint/models.py

```diff
@@ -74,13 +74,20 @@ def to_file_path(self) -> Path:
             raise ValueError(f"Failed to decode JSON from file {file_path}: {err}") from err
 
     @classmethod
-    def from_json_file(cls: type["TokenMetadata"], file_path: Path | None) -> "TokenMetadata":
+    def from_json_file(cls, file_path: Path | None, name: str | None, decimals: int | None) -> "TokenMetadata":
```

Comment:
Based on this, it appears that it's possible for name to be None, however the type suggest that it isn't?